transmart-core-db-tests/test/integration/org/transmartproject/db/ontology/OntologyTermTagsResourceServiceTests.groovy

```diff
@@ -118,13 +118,25 @@ class OntologyTermTagsResourceServiceTests {
                     hasProperty('key', equalTo(key)),
```

Comment:
Hmm, interesting. btw, I didn't write the code, only moved it. But it appears as if the `equals` method of the implementation (probably `AbstractI2b2Metadata`, using `@EqualsAndHashCode`) does accept the `String` argument, right? Maybe better would be something like `hasProperty('fullName', equalTo(...))`...